text/static/text/js/toolbar.js

```diff
@@ -22,17 +22,34 @@
 
     function toggle_toolbar() {
         if (toolbar_active) {
-            toolbar.hide();
-            handle.show();
+            toolbar.removeClass("toggle");
             body.css('overflow', 'visible');
         } else {
-            toolbar.show();
-            handle.hide();
+            toolbar.addClass("toggle");
             body.css('overflow', 'hidden');
         }
         toolbar_active = !toolbar_active;
     }
 
+    $(".djtext_submit").click(function() {
+           $("#djtext_form").submit();
+           toolbar.toggleClass("toggle");
+           body.css('overflow', 'visible');
+           $(this).hide();
+    });
+
+    var changes = 0;
```

Comment:
Move declaration to the top. This probably needs to be set to `0` in load_text :+1: